ps_sharebuttons.php

```diff
@@ -95,43 +95,43 @@ public function getContent()
         $helper->submit_action = 'submitSocialSharing';
         $helper->currentIndex = $this->context->link->getAdminLink('AdminModules', false).'&configure='.$this->name.'&tab_module='.$this->tab.'&module_name='.$this->name;
         $helper->token = Tools::getAdminTokenLite('AdminModules');
-        $helper->tpl_vars = array('fields_value' => $this->getConfigFieldsValues());
+        $helper->tpl_vars = ['fields_value' => $this->getConfigFieldsValues()];
 
-        $fields = array();
+        $fields = [];
         foreach (self::$networks as $network) {
-            $fields[] = array(
+            $fields[] = [
                 'type' => 'switch',
                 'label' => $network,
                 'name' => 'PS_SC_'.Tools::strtoupper($network),
-                'values' => array(
-                    array(
+                'values' => [
+                    [
                         'id' => Tools::strtolower($network).'_active_on',
                         'value' => 1,
-                        'label' => $this->trans('Enabled', array(), 'Admin.Global'),
-                    ),
-                    array(
+                        'label' => $this->trans('Enabled', [], 'Admin.Global'),
+                    ],
+                    [
                         'id' => Tools::strtolower($network).'_active_off',
                         'value' => 0,
-                        'label' => $this->trans('Disabled', array(), 'Admin.Global'),
-                    ),
+                        'label' => $this->trans('Disabled', [], 'Admin.Global'),
+                    ],
                 ),
             );
```

Comment:
Bad git merge

Reply:
Fixed in https://github.com/PrestaShop/ps_sharebuttons/pull/18/files#diff-3a8888c8085e5d6430bb741c68218053